evap/evaluation/tests/test_models.py

```diff
@@ -537,6 +538,21 @@ def test_textanswer_review_state(self):
             evaluation.TextAnswerReviewState.REVIEWED,
         )
 
+    def test_state_change_log_translated(self):
+        evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.NEW,
+        )
+        evaluation.save()
```

Comment:
```suggestion

Reply:
Thats probably true :+1: . The docs are also implying this